doc/configuration.md

```diff
@@ -1,4 +1,205 @@
 # Configuration
 
-**TODO(stevvooe): This should include an exhaustive account configuration
-parameters, including those for various subsystems.
\ No newline at end of file
+Below is a comprehensive example of all possible configuration options for the registry. Some options are mutually exclusive, and each section is explained in more detail below, but this is a good starting point from which you may delete the sections you do not need to create your own configuration. A copy of this configuration can be found at config.sample.yml.
+
+```yaml
+version: 0.1
+loglevel: debug
+storage:
```

Comment:
It might be good to include a copy of each yaml portion in the section that explains.